lstm.py

```diff
@@ -101,18 +115,41 @@ def add_model(self, input_data):
         return sentence_logits
 
 
+    def add_perplexity_op(self, sentence_logits):
+        # TODO is this redundant with add_loss_op? Maybe be incurring
```

Comment:
This seems a little unsatisfying... but I think that the log sum means we can't just make this a function of a scalar loss